app/support/statement_pdf_default.rb

```diff
@@ -0,0 +1,71 @@
+class StatementPdfDefault < StatementPdf
```

Comment:
Is there a reason to introduce a class hierarchy with one subclass at this point? Having a base class, and then a subclass named `*Default` seems a bit weird here.

I think the inheritance hierarchy adds complexity here, and it may be YAGNI—if we find that we need to override the behavior down the road, can we figure out how to best achieve it at that point? I’m not sure that inheritance would be the best solution even if we do need to vary behavior between schools later…

Reply:
Right now, each of the schools has a class that in one of their engines that inherits from `StatementPdf` and overrides `generate`. So while the class hierarchy does not exist in open, it already does exist in the forks. With this, I was trying to break apart the base class that the schools' custom statements rely on and the sample implementation that's included in open. Maybe `Sample` is a better name than `Default`?

Reply:
> So while the class hierarchy does not exist in open, it already does exist in the forks.

Oh, this is the piece I was missing, thanks for the context 😄

> Maybe `Sample` is a better name than `Default`?

How about `Open` to mirror the repository? Or do you think that’s too ambiguous/confusing? In either case, I like `Sample` better than `Default` so go with your ❤️!